public/app/plugins/datasource/grafana-azure-monitor-datasource/types/query.ts

```diff
@@ -51,6 +54,11 @@ export interface AzureMetricQuery {
 
   /** @deprecated Remove this once angular is removed */
   allowedTimeGrainsMs?: number[];
+
+  /** @deprecated migrated to dimensionFilters */
+  dimension?: string;
+  /** @deprecated migrated to dimensionFilters */
```

Comment:
remove this comment?

Reply:
I want the types to remain so I don't have to cast to any to access them in the migration, but I want the `@deprecated` annotation and comment so people know not to try and access it elsewhere.